src/card.css

```diff
@@ -0,0 +1,97 @@
+.store__desc {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  text-align: justify;
+  margin: 40px 16px 0;
```

Comment:
You should set correct paddings for the main card and set margin-top or margin-bottom to all child elements